index.html

```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-4.png" alt="project-snapshot">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
+            <div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+    </section>
+    <section class="about-me">
+        <div class="about-container">
+            <h1 class="about-title">About Myself</h1>
+            <div class="about-body">
+                <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <h2>LET'S CONNECT</h2>
+            </div>
+            <ul class="social-media-icons">
+                <li class="social-media-icon"><a href="https://twitter.com/ebidien83"><img src="twitter.png"
+                            alt="twitter" /></a></li>
+                <li class="social-media-icon"><a href="https://www.linkedin.com/in/ebidien"><img src="linkedin.png"
+                            alt="linkedin" /></a></li>
+                <li class="social-media-icon"><a href="https://medium.com/@ebidien"><img src="medium.png"
+                            alt="medium" /></a></li>
+                <li class="social-media-icon"><a href="https://github.com/ebidien"><img src="github.png"
+                            alt="github" /></a></li>
+                <li class="social-media-icon"><a href="https://angel.co/u/dienebi-ombu"><img src="angellist.png"
+                            alt="angellist" /></a></li>
+            </ul>
+            <button class="resume-link" type="button">Get my resume</button>
+            <section class="about-stack">
+                <div class="stack-container">
+                    <div class="stack-category">
+                        <h2>Languages</h2>
+                        <i class="fa fa-chevron-down"></i>
```

Comment:
- [ ] Please use this color `#ACB7C3` for the drop-down menu to match the design

Reply:
Corrected the color inline thus: `<i class="fa fa-chevron-down" style="color: acb7c3;"></i>`